aws-core/src/main/java/software/amazon/awssdk/awscore/internal/client/config/AwsClientOptionValidation.java

```diff
@@ -13,9 +13,11 @@
  * permissions and limitations under the License.
  */
 
-package software.amazon.awssdk.awscore.config.options;
+package software.amazon.awssdk.awscore.internal.client.config;
```

Comment:
per our conversation on chime this should be inverted. client.config.internal

Reply:
Whoops, then I did all of this wrong.